lib/storage/commands/push.rb

```diff
@@ -33,7 +33,9 @@ def run
         # ARGS
         # [ source_file, destination ]
 
-        valid_args = args.map { |a| a&.prepend("/")&.gsub(%r{/+}, "/") }
+        valid_args = args.dup
+        valid_args[1] = valid_args[1].dup.prepend("/")
+        valid_args = valid_args.map { |a| a&.gsub(%r{/+}, "/") }
```

Comment:
```
jack@jack-alces ~/code/flight-storage (dev/command_standardise) $ bin/storage push Polygondwanaland_Mp3.zip
storage: undefined method `prepend' for nil:NilClass
```

Reply:
So I only used `prepend` so I could use the safe navigation operator and somehow _that's_ what I missed